src/LogicApp/LogicApp/help/Get-AzIntegrationAccountAssembly.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
It looks like the Az.LogicApp module help file was not update - please run once with update-markdownhelpmodule so that the new cmdlets get added to the module help file.

Reply:
When I regenerated the help files last time, they all had updates due a change in encoding I think. Should I include those or just generate and revert all the ones I didn't touch (which is what I've been doing)

Reply:
Feel free to revert the ones you didn't touch - be sure to include the Az.LogicApp change though.

Reply:
Will do